src/com/content-event/event-graded-game.less

```diff
@@ -0,0 +1,23 @@
+@import "defs.less";
```

Comment:
(not this line but) This file should be named `event-my-grades`.

Reply:
Removed file